SwiftyJSON.xcodeproj/project.pbxproj

```diff
@@ -670,7 +684,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
```

Comment:
It's just not visible, but you can type 7.0 and it works correctly. I assume you've already found it out, because the fix is already pushed :)